observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-troubleshooting.adoc

```diff
@@ -0,0 +1,85 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="dist-tracing-tempo-troubleshooting"]
+= Troubleshooting
+include::_attributes/common-attributes.adoc[]
+:context: dist-tracing-tempo-troubleshooting
+
+toc::[]
+
+There are multiple ways on you can diagnistic and fix common problems with the TempoStack instance,
+
+You can inspect the logs of the different components along with some useful metrics.
+
+If the the TempoStack instance has no traces, there are two possible causes:
+* TempoStack instance is not ingesting traces.
+* There is an issue with the query path.
+
+[id="getting-tempo-stack-logs_{context}"]
+== Getting the TempoStack instance logs
+
+You can get the logs from the Tempo component as follows.
+
+.Procedure
+
+. Localize the component you want to see the logs. You can do this by listing all the deployments on an specific namespace.
+
+. Identify the pods belonging to that component.
+
+. Watch for the logs using `oc logs` command or the web console to retrieve the logs.
+
+[id="problems-ingesting-traces_{context}"]
+== Troubleshooting problems with ingesting traces
+
+Use the following steps to troubleshoot when the TempoStack instance fails to ingest traces.
+
+.Procedure
+
+. Review the components in the ingestion path to determine which problem you are facing:
++
+* The spans are not sent correctly.
+* The spans are not sampled.
+
+. Check for errors in the logs that identify unhealthy ingesters, which might be failing due to out-of-memory (OOM) issues or scale-down events.
+
+. Check the following two metrics to verify that the TempoStack instance is successfully receiving traces:
++
+[source,yaml]
+----
+tempo_distributor_spans_received_total
+tempo_ingester_traces_created_total
+----
++
+** If both of these metrics are greather than zero, the TempoStack instance is successfully ingesting data.
+** If `tempo_distributor_spans_received_total` is `0`, the distributor is not receiving spans.
+Troubleshoot by checking the network configuration, protocols, and ports of the applications that are attempting to send traces to the distributor.
+** If `tempo_ingester_traces_created_total is `0`, there is a communcation problem between the ingester and the distributor. Troubleshoot by inspecting both logs to find errors.
+
+[id="reviewing-ingestion-limits_{context}"]
+=== Reviewing ingestion limits
+
+In high-volume tracing environments, the default trace limits might not be adequate.
+If spans are being refused due to these limits, you will see logs like this at the distributor:
+// (logs like this at the distributor...) indicating some limits are passed
+
+You will also see the `tempo_discarded_spans_total` metric incremented.
```

Comment:
@rubenvp8510, could you please clarify this in reply in this comment thread? Is the metric incremented a symptom of something? How is it incremented?

Reply:
I've moved this line in the file, so I'll repost this question as a new comment.